vital/algo/detected_object_set_input.h

```diff
@@ -113,8 +123,13 @@ class VITAL_ALGO_EXPORT detected_object_set_input
 };
 
 /// Shared pointer type for generic detected_object_set_input definition type.
-typedef std::shared_ptr<detected_object_set_input> detected_object_set_input_sptr;
+typedef std::shared_ptr< detected_object_set_input >
+  detected_object_set_input_sptr;
+
+} // namespace algo
+
+} // namespace vital
 
-} } } // end namespace
+}     // end namespace
```

Comment:
```suggestion
} // namespace kwiver
```
Other instances exist; I'm not going to individually flag them. (One of these days I'd really like to teach uncrustify to forcibly remove existing namespace comments...)